docs/changelog.md

```diff
@@ -0,0 +1,88 @@
+---
+id: Changelog
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+# Changelog
+
+## October 6, 2023
+
+### Breaking Changes (Code Incompatibility)
+
+❗[responsive-pub/134](https://github.com/responsivedev/responsive-pub/pull/134) 
+and [responsive-pub/135](https://github.com/responsivedev/responsive-pub/pull/135)
+changed the way a new instance of `ResponsiveKafkaStreams` is created. These
+changes apply to `dev.responsive.kafka-client` versions `0.12.0` and above.
```

Comment:
This is kind of easy to miss, should we have an actual sub-header under breaking changes to specify the module (like we do eg down below with `### Kafka Client (0.12.0)` )

alternatively, we could break it up by module first, then have a sub-section for breaking changes at the top of the section for each module (if there are any). Personally, this organization makes more sense to me, as people are likely to be upgrading and thinking about each module independently. I'll leave this up to you to decide though